### ISSUE TITLE
startpos switcher bugs

### ISSUE THREAD

**Issue by Matistan** [TheSillyDoggo/GeodeMenu#26]
Bugs I found in startpos switcher:
- when entering a lvl with n start positions, it shows <0 / n> instead of <n / n>, and I sometimes start not from the last start pos, but from the last placed startpos
- before an update from few hours ago, the keybinds 'q' and 'e' worked for switching start positions, now they don't, so the only way to switch start pos is to click the triangles with an invisible mouse :/

(this is not a bug bug a feature request) can you pls add smart start pos? (automatically changes your gravity, gamemode, speed, etc. based on the last gamemode portal, speed change, gravity portal, etc.)

**Comment by FRareDomm** [comment]
yup, same issue here, but also i wanna say when this bug is fixed, can u also add keybinds to change the startpos, cuz i would rather use arrow keys rather than q and e to change startpos, just like the startpos switcher mod from b4 in 2.204
